src/consumer/views/view.jsx

```diff
@@ -219,26 +219,31 @@ export default function ConsumerView(props) {
             value="csv"
           />
 
-          {/* commenting out for now until https://marvellconsulting.atlassian.net/browse/SW-581 is completed */}
-          {/* <RadioGroup
+          <RadioGroup
             name="number_format"
```

Comment:
You've updated the controller to 
```
const view = req.query.view_choice;
```
but the name submitted for the radio form is still "number_format" - does it actually work?